Cargo.toml

```
@@ -364,6 +364,7 @@ userenv = []
 usp10 = []
 utilapiset = []
 uxtheme = []
+versrc = []
```

```suggestion
verrsrc = []
```

Looks like this typo happened everywhere.